### PR TITLE
Create DevSetup-QuickGuide

### PR DESCRIPTION
**Pull request by vgonzalez9**
New Developer Setup guide.

NOTE: Want to mitigate duplicate information, will need to clean up other README file to reference this as part of the initial setup process.

### CODE REVIEW

File: docs/DevSetup-QuickGuide [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+export RDW_REPORTING_KEYSTORE_ENTRY_PASSWORD=mypassword
+export RDW_REPORTING_IDP=https://sso-deployment.sbtds.org/auth/saml2/jsp/exportmetadata.jsp?realm=/sbac
+export RDW_REPORTING_ENTITYID=rdw-reporting-ui-local
+export RDW_AWS_ACCESS_KEY=myAccessKey
+export RDW_AWS_SECRET_KEY=mySecretKey
+export RDW_RESOURCE_ROOT=s3://rdw-resources/
+export RDW_PERMISSION_SERVICE=http://perm-web-deployment.sbtds.org:8080/rest
+export RDW_REPORTING_IRIS_VENDORID=2B3C34BF-064C-462A-93EA-41E9E3EB8333
+export RDW_REPORTING_IRIS_URL=http://iris-dev.sbacdw.org:8080/iris/
+export RDW_REPORTING_ARTIFACTS_EXAM_ITEM_DIRECTORY=item-content/bank/items/Item-{0}/
```
Comment (malaffoon) [inline docs/DevSetup-QuickGuide:92]: i know this is existing documentation but can we come up with an approach that doesn't use environment variables?

Comment (gregcharles) [reply docs/DevSetup-QuickGuide:92]: With the exception of GIT_USER and GIT_PASSWORD, all of these values could come from rdw-config, couldn't they? The only one I could see devs wanting to override is the keystore path one (and ~/Downloads/rdw-reporting-saml.jks isn't the default I would go with).

Comment (malaffoon) [reply docs/DevSetup-QuickGuide:92]: I'm not sure how running from node affects things. These are settings for the underlying spring boot service which is, i assume, still talking to the config server, even when you are running the angular part via node. 

One thing we did in TIMS is make it so you could run all the services locally as a developer without the config service/repo. I wonder if we want to burn a couple days getting that working. Hmm.

**Comment by vgonzalez9** [comment]
Closing.
Moving this to the RDW_Reporting project with a new PR.

**Pull request closed without merging**